packages/app/src/store/runner-ui-store.ts

```diff
@@ -40,6 +41,7 @@ export const useRunnerUiStore = defineStore({
       reporterWidth: runnerConstants.defaultReporterWidth,
       automationStatus: automation.CONNECTING,
       randomString: `${Math.random()}`,
+      noCommandLog: window.__CYPRESS_CONFIG__.noCommandLog,
```

Comment:
I find it's best to name vars with words that have read with true/value meanings. For example `showCommandLog` if true, it shoes, if false, it hides it or hasCommandLog . Can we rename noCommandLog to something that reads like the truthy / falshiness of the value?

Reply:
I agree in general - I kept it like this to not lose the connection to the actual name of the environment variable, but the runner UI short is probably the best place to break that connection, and convert it from "information in the config that came from the CLI" to a more readable boolean name, will update.

Reply:
I'm going with `hideCommandLog` every where to at least keep the truthy/falsy patten matched with `CYPRESS_NO_COMMAND_LOG=1`. I think it does read better this way.

Reply:
I have a few reasons I like it in the store. This global variable stuff is a weird little exception for values that get used in run mode and can't come out of gql - I'd rather the components no have to know/care about that. It feels weird to grab it from the window in the Open Mode runner component, but also weird to have 2 slightly different ways to get the same data depending on which mode we are in. Putting it in the store, both components who need to use this value can get it in the same way and not care about the source, and both components will update correctly over time if something about the source changes.

I don't feel super strongly about this, but I think it's better than the alternatives.